tests/integration/pilot/mcp/testdata/kube/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    deployment.kubernetes.io/revision: "1"
+  generation: 1
+  labels:
+    app: hello-node
+  name: hello-node
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: hello-node
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: hello-node
+    spec:
+      containers:
+      - image: gcr.io/hello-minikube-zero-install/hello-node
```

Comment:
We shouldn't be starting arbitrary images in the tests here. Can we just the echo image? Some folks are running these offline and ideally we don't depend on external images

Reply:
oof yeah, good catch ... I missed this.